### ISSUE TITLE
Metadata For OutputVariable Tables?

### ISSUE THREAD

**Issue by aruzicka555** [LANDIS-II-Foundation/Extension-Output-Biomass-PnET#3]
Each OutputVariable in the extension has a table output in addition to a map output - either a species or ecoregion table. Are these table outputs something we want metadata to track?

**Comment by rmscheller** [comment]
I think the answer is yes.  Brian?  Please answer and we will return to issue #1

**Comment by brmiranda** [comment]
Yes, the table outputs are important to include in the metadata.
